test/testrunner.py

```diff
@@ -179,9 +189,17 @@ def main():
                             _resp, keyname = destroy_ssh_keypair(ec2_client, key_file)
                             LOGGER.info(f"Deleted {keyname}")
     elif specific_test_type == "sagemaker":
-        run_sagemaker_tests(
-            [image for image in standard_images_list if not ("tensorflow-inference" in image and "py2" in image)]
-        )
+        if benchmark_mode:
+            report = os.path.join(os.getcwd(), "test", f"{test_type}.xml")
+            os.chdir(os.path.join("test", "dlc_tests"))
+
+            setup_sm_benchmark_env(dlc_images, test_path)
+            pytest_cmd = ["-s", "-rA", test_path, f"--junitxml={report}", "-n=auto", "-o", "norecursedirs=resources"]
```

Comment:
"-o norecursedirs" ensures that pytest does not recursive execute the tests in any clones of other GitHub repositories that have been done for the sake of the tests.